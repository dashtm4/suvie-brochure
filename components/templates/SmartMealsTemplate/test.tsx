import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { SmartMealsTemplate } from '.'

describe('<SmartMealsTemplate />', () => {
  it('should renders correct <SmartMealsTemplate />', () => {
    const wrapper = shallow(<SmartMealsTemplate />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
