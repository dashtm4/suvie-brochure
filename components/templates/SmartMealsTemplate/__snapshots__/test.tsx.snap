// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SmartMealsTemplate /> should renders correct <SmartMealsTemplate /> 1`] = `
<main>
  <FeaturesBanner
    features={
      Array [
        Object {
          "image": Object {
            "height": 40,
            "url": "/images/affirm-logo.svg",
            "width": 40,
          },
          "title": "Pay as low as $33/ month",
        },
        Object {
          "image": Object {
            "height": 40,
            "url": "/images/100-day-home-trial-risk-free.svg",
            "width": 40,
          },
          "link": "/free-trial/",
          "title": "100-day home trial, risk-free",
        },
        Object {
          "image": Object {
            "height": 30,
            "url": "/images/logo-seal-black.svg",
            "width": 30,
          },
          "title": "1 Year Warranty",
        },
      ]
    }
  />
  <DefaultHero
    imageSrc="/images/smart-meals-page/smart-meals-hero-background.jpg"
    mobileImageSrc="/images/smart-meals-page/smart-meals-hero-background--small.jpg"
    title="Smart Meals"
  />
  <SmartMealsSecondSection />
  <SmartMealsThirdSection
    mealsListRef={
      Object {
        "current": null,
      }
    }
  />
  <SmartMealsForthSection
    mealsListRef={
      Object {
        "current": null,
      }
    }
  />
  <SmartMealsFifthSection />
  <FinalCallToActionSection
    imageSrc="/images/smart-meals-page/final-call-to-action-section-background.jpg"
    offsetX="left"
    offsetXInMobile="center"
  />
</main>
`;
